app/src/main/java/com/conupods/OutdoorMaps/View/MainActivity.java

```diff
@@ -16,59 +16,50 @@
 
 public class MainActivity extends AppCompatActivity {
 
-
     private static final String TAG = "MainActivity";
     private static final int ERROOR_DIALOG_REQUEST = 9001;
     Button MapActivityButton;
 
-
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        if(isGoogleAPIServiceAvailable())
+        if (isGoogleAPIServiceAvailable())
             init();
 
 
     }
 
-
-    private void init(){
+    private void init() {
         MapActivityButton = (Button) findViewById(R.id.MapActivityButton);
         MapActivityButton.setOnClickListener(new View.OnClickListener() {
 
             @Override
-            public void onClick(View view){
+            public void onClick(View view) {
                 startActivity(new Intent(MainActivity.this, MapsActivity.class));
 
             }
         });
     }
 
-
-    private boolean isGoogleAPIServiceAvailable(){
+    private boolean isGoogleAPIServiceAvailable() {
         Log.d(TAG, "Checking is Google API services are available...");
         Boolean GoogleAPIServiceAvailable = false;
 
         int available = GoogleApiAvailability.getInstance().isGooglePlayServicesAvailable(MainActivity.this);
```

Comment:
I think the code would be more readable and versatile if this was broken down. It would also avoid redundancy of `GoogleApiAvailability.getInstance()`.

e.g.
`GoogleApiAvailability api = GoogleApiAvailability.getInstance();`
 `int serviceAvailable= api.isGooglePlayServicesAvailable(this);`

Reply:
yessir, fully agree